tests/user/test_get_user_by_id.py

```diff
@@ -0,0 +1,203 @@
+import pytest
+from allure import feature, description, link, step, title
+from hamcrest import assert_that, is_, contains_string, is_not, empty
+
+from framework.asserts.common import assert_status_code, assert_content_type, assert_response_message
+from framework.asserts.user_asserts import assert_all_user_data_matches
+from framework.endpoints.authenticate_api import AuthenticateAPI
+from framework.endpoints.users_api import UsersAPI
+from framework.tools.generators import generate_user
+from framework.tools.generators import generate_jwt_token
+
+
+@feature("Getting user info by ID")
+class TestGetUserById:
+    @title("Getting User's Own Information by ID")
+    @description(
+        "GIVEN the user is logged in, "
+        "WHEN the user sends a request to get their own information by ID, "
+        "THEN the response code is 200 and the response body contains the current user's data."
+    )
+    def test_get_user_self_info_with_valid_id(self, create_authorized_user):
+        user, token = [create_authorized_user["user"], create_authorized_user["token"]]
```

Comment:
`with step`?

Reply:
Thanks for the review and the idea to use an Allure step for the line user, token = [create_authorized_user["user"], create_authorized_user["token"]].

I think this line is just a simple setup step and not a major part of the test. Adding it as an Allure step might make our reports too detailed and less clear. What's your opinion on this?